tromega/lib/views/home_view.dart

```diff
@@ -1,17 +1,185 @@
+import 'package:flutter/gestures.dart';
 import 'package:flutter/material.dart';
+import 'package:tromega/views/traindayview.dart';
 import '../widgets/bottom_menu.dart';
+import '../views/social_space_view.dart';
+import '../views/plans/plan_overview.dart';
+import 'package:calendar_timeline/calendar_timeline.dart';
 
-class HomeView extends StatelessWidget {
+class HomeView extends StatefulWidget {
   const HomeView({Key? key}) : super(key: key);
 
+  @override
+  State<HomeView> createState() => _HomeViewState();
+}
+
+class _HomeViewState extends State<HomeView> {
+  late DateTime dateToday;
+  late Color colorDot;
+
+  @override
+  void initState() {
+    dateToday = DateTime.now();
+    colorDot = Color.fromARGB(1000, 4, 146, 240);
+    super.initState();
+  }
+
+  var trainigsDaten = Trainweek("week", [
+    Trainday(DateTime.now().subtract(const Duration(days: 1)), true),
+    Trainday(DateTime.now().subtract(const Duration(days: 2)), false),
+    Trainday(DateTime.now().subtract(const Duration(days: 3)), true),
+    Trainday(DateTime.now().subtract(const Duration(days: 4)), false),
+    Trainday(DateTime.now().subtract(const Duration(days: 5)), false),
+    Trainday(DateTime.now().subtract(const Duration(days: 6)), false),
+    Trainday(DateTime.now().subtract(const Duration(days: 7)), true)
+  ]);
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: AppBar(title: const Text('Hier Icon einfügen')),
-      body: const Text('Maiks\' ToDo'),
-      bottomNavigationBar: const BottomMenu(
-        index: 0
+      appBar: AppBar(
+        backgroundColor: Color.fromARGB(1000, 0, 48, 80),
+        bottom: PreferredSize(
+          preferredSize: const Size.fromHeight(120.0),
+          child: Column(children: [
+            Container(
```

Comment:
Padding(
              padding: const EdgeInsets.all(8.0),
              child:

Reply:
Also dann von Z44 bis Z91 alles in Padding()? @steffenhls

Reply:
yes